CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.1.0 (2019-03-19)
+
+- Added `ariaLabel` property to radio group components. [#26](https://github.com/blackbaud/skyux-forms/pull/26)
+- Fixed radio group component to properly watch for changes to inner radio components. [#26](https://github.com/blackbaud/skyux-forms/pull/26)
```

Comment:
I'd delete "properly" because it suggests we were doing this wrong before and I don't think that's the case. And I'd change "inner radio components" to "its radio buttons."

Reply:
It was wrong before :(

Reply:
I changed the other wording

Reply:
Sounds good.